finagle-thrift/src/main/thrift/tracing.thrift

```diff
@@ -48,7 +48,7 @@ struct Annotation {
   1: i64 timestamp                 // microseconds from epoch
   2: string value                  // what happened at the timestamp?
   3: optional Endpoint host        // host this happened on
-  4: optional i32 duration         // how long did the operation take? microseconds
+  // 4: optional i32 OBSOLETE_duration         // how long did the operation take? microseconds
```

Comment:
i generally prefer leaving the fields, but no big deal.

can you explicitly state that this field number cannot be reused?